platform/features/listview/bundle.js

```diff
@@ -0,0 +1,42 @@
+define([
```

Comment:
Our sources should start with a comment containing licensing info; you can copy that over from other JS/HTML files. This applies here and to other new files.